### PR TITLE
Add and re-prioritize icons in url_icon

### PR DESCRIPTION
**Pull request by kernc**
This PR adds some new URL icons for project URLs:
* [`fa-bug`](https://fontawesome.com/icons/bug?style=solid) for issue tracker,
* [`fa-coins`](https://fontawesome.com/icons/coins?style=solid) for donations link.

The trigger strings are those recommended in specs for `project_urls` ([PEP-459](https://www.python.org/dev/peps/pep-0459/#project-urls), [pypa/sampleproject/setup.py](https://github.com/pypa/sampleproject/blob/559a84bd6ecbd6b8cd02a105b1f8933956b3f7e7/setup.py#L194-L197)).

Additionally, homepage (`fa-home`) and documentation (`fa-book`) icons are reordered so that they are preferred over more generic service (e.g. GitHub) icons.

### CODE REVIEW

File: warehouse/templates/packaging/detail.html [hunk 1]
```diff
@@ -18,7 +18,15 @@
...
-   {% set icon = "fa fa-cloud-download-alt" %}
+   {% set icon = "fas fa-cloud-download-alt" %}
+{% elif name|lower in ["home", "homepage", "home page"] %}
+  {% set icon = "fas fa-home" %}
+{% elif name.lower().startswith(("docs", "documentation")) or parsed.netloc in ["readthedocs.io", "readthedocs.org", "rtfd.io", "rtfd.org"] or parsed.netloc.endswith((".readthedocs.io", ".readthedocs.org", ".rtfd.io", ".rtfd.org")) or parsed.netloc.startswith(("docs.", "documentation.")) %}
+  {% set icon = "fas fa-book" %}
+{% elif name.lower().startswith(("bug", "issue", "tracker", "report")) %}
+  {% set icon = "fas fa-bug" %}
+{% elif name.lower().startswith(("funding", "donate", "donation")) %}
+  {% set icon = "fas fa-coins" %}
```
Comment (ewdurbin) [inline warehouse/templates/packaging/detail.html]: coins look a bit too much like the generic "database" icon, I think I'd personally prefer [`fas fa-donate`](https://fontawesome.com/icons/donate?style=solid)

Comment (kernc) [reply warehouse/templates/packaging/detail.html]: Of course!

**Comment by ewdurbin** [comment]
Thanks @kernc!

**Comment by kernc** [comment]
Thank you!

**Pull request merged**